sdk/webpubsub/Azure.Messaging.WebPubSub/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0001:Use one of the following pre-approved namespace groups (https://azure.github.io/azure-sdk/registered_namespaces.html): Azure.AI, Azure.Analytics, Azure.Communication, Azure.Data, Azure.DigitalTwins, Azure.Iot, Azure.Learn, Azure.Media, Azure.Management, Azure.Messaging, Azure.Search, Azure.Security, Azure.Storage, Azure.Template, Azure.Identity, Microsoft.Extensions.Azure", Justification = "<Pending>", Scope = "namespace", Target = "~N:Azure.Rest.WebPubSub")]
+[assembly: SuppressMessage("Usage", "AZC0008:ClientOptions should have a nested enum called ServiceVersion", Justification = "REST clients don't have custom options", Scope = "type", Target = "~T:Azure.Rest.WebPubSub.WebPubSubServiceRestClientOptions")]
```

Comment:
I've never seen this happen for rest clients before.  I wonder if it's a new bug in the generated code or a new bug in the analyzer?  +@pakrym?

Reply:
No, this is a leftover from my LLC experiments.